README.md

```diff
@@ -41,23 +62,35 @@ Apparently the Streamlabs API had a rate limit. Meaning you could only do 20 cal
 I've then decided this had to be fixed and created a proxy server system in order to prevent rate-limiting to occur. The website still calls 1 API, but the API then calls one of the proxy servers (and kept track of which one was rate-limited and which wasn't), and the proxy server in turn calls the Streamlabs API to convert the message.
 
 This setup still works to this day, but having to host more and more proxies as the popularity increases is not a working solution for me. I've had the idea for a few months now, but finally made a desktop application for Elundus Core which comes with it's own bundled API which calls Streamlabs, which means I don't have to host an API or proxyservers anymore.
+</details>
 
-## I can't play the downloaded TTS sound
+<details>
+  <summary><b>I can't play the downloaded TTS sound</b></summary>
+  <br />
 The Streamlabs sound file is in the [Ogg Vorbis](https://en.wikipedia.org/wiki/Vorbis) format. You can download a codec to open it in your favorite music player/editing software. I didn't have much luck myself using the codec, so I used a converter to convert the file to MP3 so you don't have to install a codec. Sadly I can't convert the file in the application itself.
 
 Examples of converters are:
 - [FlicFlac](https://github.com/DannyBen/FlicFlac) (open-source application)
 - [Convertio](https://convertio.co/oga-mp3/) (online)
 - [Cloudconvert](https://cloudconvert.com/oga-to-mp3) (online)
+</details>
+
+---
 
 # Contributing
-Requirments for developing are:
-- NodeJS
-- Yarn (```npm i -g yarn```)
-- Electron installed globally using NPM (```npm i -g electron```)
+To download the project source code and start developing :
```

Comment:
There's an extra space before the colon. Would look more clean

Reply:
Should be fixed now